Pronto-MIA/Pronto-MIA/BusinessLogic/API/Mutation.cs

```diff
@@ -74,7 +75,7 @@ public class Mutation
         /// </exception>
         [Authorize]
         [UseSingleOrDefault]
-        public async Task<IQueryable<DeploymentPlan?>> UpdateDeploymentPlan(
+        public async Task<IQueryable<DeploymentPlan>> UpdateDeploymentPlan(
             [Service] IDeploymentPlanManager deploymentPlanManager,
             int id,
             IFile? file,
```

Comment:
Why is file defined as a nullable here?

Reply:
Because you do not have to update the file. You may send an update which only adjusts the description for example.